src/protocolsupport/protocol/typeremapper/pe/PEDataValues.java

```diff
@@ -137,9 +140,177 @@ public static NetworkEntityType getLivingTypeFromPeNetworkId(int networkId) {
 	}
 
 	public static final ArrayBasedIdRemappingTable BLOCK_ID = new ArrayBasedIdRemappingTable(MinecraftData.BLOCK_ID_MAX * MinecraftData.BLOCK_DATA_MAX);
+
+	private static HashMap<Integer, ArrayList<Integer>> blockIdRemaps = new HashMap<>();
```

Comment:
Follow LSP.

Reply:
We don't require following LSP actually, but this field is actually missing an final modifier, which is more important IMO.